mkosi

```diff
@@ -1784,6 +1785,61 @@ gpgkey={gpg_key}
 
     reenable_kernel_install(args, root, masked)
 
+@completestep('Installing OpenMandriva')
+def install_openmandriva(args: CommandLineArguments, root: str, do_run_build_script: bool) -> None:
+    release = args.release.strip("'")
+    masked = disable_kernel_install(args, root)
+
+    # OpenMandriva does not (yet) have RPM GPG key on the web
+    gpg_key = '/etc/pki/rpm-gpg/RPM-GPG-KEY-OpenMandriva'
+    if os.path.exists(gpg_key):
```

Comment:
Should consider the scenario here where the key is not found. You can define a new exception and catch it in main alongside the two other ones and print an error.

Reply:
Yes I know about that, but public keys are needed for rpm package. So if this file is missing then it means things are heavily broken. I assume that scenario never happens on purpose.

Reply:
It's possible that we'd want to build openmandriva packages on other distros where this key might not be available. In that case, we'd crash somewhere further down with maybe a not as clear error. If we check here, we get a clear error that there's a missing key that we need which will be much more user friendly than crashing with an unhandled python exception further down.

Reply:
anyways i'll add some error message

Reply:
i've added die() with proper message.